[2024-12-06T00:24:54.901+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T00:24:55.672+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: class_act_7_pipeline.collect_data scheduled__2024-10-15T12:00:00+00:00 [queued]>
[2024-12-06T00:24:55.974+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: class_act_7_pipeline.collect_data scheduled__2024-10-15T12:00:00+00:00 [queued]>
[2024-12-06T00:24:56.035+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-06T00:24:59.300+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): collect_data> on 2024-10-15 12:00:00+00:00
[2024-12-06T00:24:59.707+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2067) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T00:24:59.744+0000] {standard_task_runner.py:72} INFO - Started process 2218 to run task
[2024-12-06T00:24:59.783+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'class_act_7_pipeline', 'collect_data', 'scheduled__2024-10-15T12:00:00+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/***_pipeline.py', '--cfg-path', '/tmp/tmpaukwztqw']
[2024-12-06T00:24:59.797+0000] {standard_task_runner.py:105} INFO - Job 257: Subtask collect_data
[2024-12-06T00:25:03.096+0000] {task_command.py:467} INFO - Running <TaskInstance: class_act_7_pipeline.collect_data scheduled__2024-10-15T12:00:00+00:00 [running]> on host cff371549a6e
[2024-12-06T00:25:05.422+0000] {job.py:229} INFO - Heartbeat recovered after 14.10 seconds
[2024-12-06T00:25:07.284+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='abdulrahmanbutt7273@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='class_act_7_pipeline' AIRFLOW_CTX_TASK_ID='collect_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-15T12:00:00+00:00'
[2024-12-06T00:25:07.304+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-06T00:25:07.319+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-06T00:25:07.329+0000] {logging_mixin.py:190} INFO - Current task name:collect_data state:running start_date:2024-12-06 00:24:55.777124+00:00
[2024-12-06T00:25:07.344+0000] {logging_mixin.py:190} INFO - Dag name:class_act_7_pipeline and current dag run status:running
[2024-12-06T00:25:07.361+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T00:25:09.645+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T00:25:09.998+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T00:25:10.062+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=class_act_7_pipeline, task_id=collect_data, run_id=scheduled__2024-10-15T12:00:00+00:00, execution_date=20241015T120000, start_date=20241206T002455, end_date=20241206T002510
[2024-12-06T00:25:10.447+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-06T00:25:10.494+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-06T00:25:10.525+0000] {logging_mixin.py:190} INFO - Dag name:class_act_7_pipeline queued_at:2024-12-05 00:23:46.974091+00:00
[2024-12-06T00:25:10.557+0000] {logging_mixin.py:190} INFO - Task hostname:cff371549a6e operator:PythonOperator
[2024-12-06T00:25:10.770+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T00:25:11.168+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-06T00:26:04.991+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T00:26:05.158+0000] {taskinstance.py:2603} INFO - Dependencies not met for <TaskInstance: class_act_7_pipeline.collect_data scheduled__2024-10-15T12:00:00+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2024-12-06T00:26:05.328+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
